playbooks/templates/.github/workflows/tft.yml

```diff
@@ -187,8 +187,16 @@ jobs:
           tf_scope: private
           api_key: ${{ secrets.TF_API_KEY_RH }}
           update_pull_request_status: false
+{%- endraw +%}
+{%- if inventory_hostname == "network" %}
+{%- raw %}
+          tmt_hardware: '{ "memory": ">= ${{ needs.prepare_vars.outputs.memory }} MB", "network": [{"type": "eth"},{"type": "eth"}]'
```

Comment:
I think this is ok for now - but it would be nice if there were some sort of configuration file (like provision.fmf) that would be in the role code.

Reply:
I've asked about this above. This file exists but I cannot find syntax to populate it. Workflow reads memory from this file already, it would be ideal to read nics from it too. Do you know if specifying two NICs can be done in tests/provision.fmf without breaking anything?

Reply:
Here is an example from the timesync role - https://github.com/linux-system-roles/timesync/blob/main/tests/provision.fmf

```yaml
standard-inventory-qcow2:
  qemu:
    net_nic:
      # Use qemu-system-x86_64 -net nic,model=help for a list of available devices.
      model: e1000
```

which seems a different use case - so I don't think we can use provision.fmf for this.

But that is ok - we don't need to worry about this now - but at some point in the future if we need to add something similar to another role, we may need to use some sort of config file.